src/sage/misc/all.py

```diff
@@ -40,8 +40,7 @@
 lazy_import('sage.misc.dist', 'install_scripts', deprecation=34259)
 lazy_import('sage.misc.trace', 'trace', deprecation=34259)
 lazy_import('sage.misc.package', ('installed_packages', 'is_package_installed',
-                                  'standard_packages', 'optional_packages',
-                                  'experimental_packages', 'package_versions'),
+                                  'package_versions'),
```

Comment:
I suggest we remove `lazy_import('sage.misc.package',...)` altogether. Those functions are always explicitly imported and only in limited context (setup.py and feature.pkg_systems.py). `package_versions` is never ever called anywhere anymore which suggests it should be at least deprecated if not removed.

Reply:
The import is already deprecated, but it's short of 1 year of deprecation

Reply:
Fair enough.